src/components/inputs/Selector.tsx

```diff
@@ -419,3 +422,37 @@ const SelectorButton = ({
     </Button>
   )
 }
+
+interface ListComponentProps extends Omit<ScrollbarProps, 'ref'> {
+  children: React.ReactNode
+}
+
+const ListComponentInternal = forwardRef<Scrollbars, ListComponentProps>((props, ref) => {
+  const { children, ...comProps } = props
+
+  return (
+    <Scrollbars
+      {...comProps}
+      renderThumbVertical={(scrollbarProps) => (
+        <div
+          {...scrollbarProps}
+          className="thumb"
+          style={{
+            borderRadius: '6px',
+            backgroundColor: 'rgba(0, 0, 0, 0.5)',
+          }}
+        />
+      )}
+      autoHeight
+      autoHide
+      autoHideTimeout={500}
+      autoHideDuration={500}
+      hideTracksWhenNotNeeded
+    >
+      {children}
+    </Scrollbars>
+  )
+})
```

Comment:
We can use the `CustomScrollbar` here by making the children prop as optional. So that,
`if (!children) return <Scrollbars ... />`